src/tabpfn/classifier.py

```diff
@@ -722,9 +731,24 @@ def predict_logits(self, X: XType) -> np.ndarray:
         return logits_tensor.float().detach().cpu().numpy()
 
     @config_context(transform_output="default")  # type: ignore
+    @track_model_call(model_method="predict", param_names=["X"])
     def predict_proba(self, X: XType) -> np.ndarray:
         """Predict the probabilities of the classes for the provided input samples.
 
+        This is a wrapper around the `_predict_proba` method.
+
```

Comment:
```suggestion
```
I suggest we don't need this as it doesn't add anything to the public documentation, and can very quickly be observed from reading the method implementation.